src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -47,7 +65,7 @@ class ReportsBox extends React.Component {
 
     return (
       <>
-        {isLoading ? (
+        {isLoading || this.state.visits === null ? (
```

Comment:
Could you change it in order to use only one flag?

I have no issues with any of these:

* `isLoading` is very explicit, but maybe is redundant.
* `visits === null` is not so explicit, but we could understand _null_ as _unknown_, the value is _unknow_ when we are _loading_ the data, so, it is also valid.